Automattic/jetpack
# Blocks: Make webpack.config.extensions.js extend calypso-build's

#### Changes proposed in this Pull Request:

- Bump `@automattic/calypso-build` to `1.0.0-alpha.2`, which includes a basline `webpack.config.js`.
- Base `webpack.config.extensions.js` on that.

#### Testing instructions:

```
yarn distclean
yarn build-extensions
```

Verify that `index.json` is copied to `_inc/blocks/`, that there are `.deps.json` files listing dependencies, and that blocks are still working.

#### Proposed changelog entry for your changes:

Blocks: Make `webpack.config.extensions.js` extend `@automattic/calypso-build`'s
